package/expo-package/src/handlers/oniOS14LibrarySelectionChange.ts

```diff
@@ -0,0 +1,20 @@
+import { Platform } from 'react-native';
+
+import * as MediaLibrary from 'expo-media-library';
+
+const isAboveIOS14 = Platform.OS === 'ios' && parseInt(Platform.Version as string, 10) >= 14;
+
+export function oniOS14LibrarySelectionChange(callback: () => void): { unsubscribe: () => void } {
```

Comment:
Can we name it something around Camera/Media? Because `oniOS14LibrarySelectionChange` can confuse users/developers on what to use it for, for the camera or some other functionality.

Maybe `oniOS14LibrarySelectionChangeForGallery`, if it makes sense.

Reply:
Thanks makes sense to change